toolbox.py

```diff
@@ -396,3 +405,55 @@ def identifyObtrusiveness(blockStr):
     if blockStr in lookup.OBTRUSIVE:
         return 3
     return 4
+
+
+def visualizeHeightmap(*arrays, title=None, autonormalize=True):
+    """**Visualizes one or multiple numpy arrays**.
+
+    Args:
+        title (str, optional): display title. Defaults to None.
+        autonormalize (bool, optional): Normalizes the array to be
+            between 0 (black) and 255 (white). Defaults to True.
+    """
+    def normalize(array):
+        """**Normalize the array to contain values from 0 to 1**."""
+        return (array - array.min()) / (array.max() - array.min())
+
+    for array in arrays:
+        if autonormalize:
+            array = (normalize(array) * 255).astype(np.uint8)
+
+        plt.figure()
+        if title:
+            plt.title(title)
+        plt_image = cv2.cvtColor(array, cv2.COLOR_BGR2RGB)
+        plt.imshow(plt_image)
+    plt.show()
+
+
+# ========================================================= converters
+# The 'data types' commonly used in this package are:
+#
+# facing: any of 'north', 'east', 'south', 'west' (can be a sequence)
+# rotation: value from 0-15 representing number of clockwise increments of pi/4
+#
+# axis: 'x', 'y', 'z'
+# vector: any multiple of (1, 1, 1) e.g. (0, 1, -4)
+# =========================================================
+
+
+def facing2rotation(facing):
+    reference = {'north': 0, 'east': 4, 'south': 8, 'west': 12}
+    if len(facing) == 1:
+        rotation = reference[lookup.INVERTDIRECTION[facing[0]]]
+    else:
```

Comment:
So why exactly is 'facing' a sequence? Is it to support things like north-west?
Also a normal average doesn't work for angles, because they wrap around. I assume that's why you added the extra checks at the bottom? It seems really hacky. 
What if a user plops in ['north','south'] or ['north','west','west']?
Also even if you implemented the average correctly, it probably still wouldn't work correctly for things like 'north north west' as it's commonly understood in geography (half way between north and northwest)

Reply:
Created issue #51 to deal with this